Makefile

```
@@ -39,10 +39,7 @@ build/pyodide.asm.js: \
 	src/core/pyproxy.o \
 	src/core/python2js_buffer.o \
 	src/core/python2js.o \
-	src/pystone.py \
-	src/_testcapi.py \
-	src/_testinternalcapi.py \
-	src/webbrowser.py \
+	$(wildcard src/lib/**/*) \
```

Maybe,
```suggestion
	$(wildcard src/lib/py/*.py) \
```
to be a bit more specific?

---

Oops what I had was just wrong.